server/backend/api/app/routers/router_guards.py

```diff
@@ -246,6 +301,48 @@ async def get_target_user_from_id(
             return None
 
 
+async def authorize_by_user_type(
+    request: Request,
+    requester: Requester = Depends(get_current_requester),
+    target_user: Union[TargetUser, None] = Depends(get_target_user_from_id),
+):
+    """
+    This authorize_by_user_type method is used to forbidden business bceid user manage idir user's access
+    """
+    if requester.user_type_code == UserType.BCEID:
+        target_user_type_code = None
+        if target_user:
+            # in the case of granting/removing access to an existing user
+            target_user_type_code = target_user.user_type_code
+        else:
+            # in the case of granting access to a new user
+            try:
+                rbody = await request.json()
+                target_user_type_code = rbody["user_type_code"]
+            except json.JSONDecodeError:
+                target_user_type_code = None
```

Comment:
Same as previous `try-except JSONDecodeError` comment. 
We could just safely assume the request body can be decoded safely, and you are able to retrieve it from the "user_type_code" attribute or None. If it cannot decode the request "body" then something is going wrong, we don't handle it.